### PR TITLE
Task/numbers site/sippeers

### PR DESCRIPTION
**Pull request by safronenko23**
/accounts/{accountId}/sites/{siteId}
/accounts/{accountId}/sites/{siteId}/sippeers
/accounts/{accountId}/sites/{siteId}/sippeers/{sippeerId}
/accounts/{accountId}/sites/{siteId}/sippeers/{sippeerId}/tns
/accounts/{accountId}/sites/{siteId}/sippeers/{sippeerId}/tns/{tn}
/accounts/{accountId}/sites/{siteId}/sippeers/{sippeerId}/totaltns
/accounts/{accountId}/sites/{siteId}/sippeers/{sippeerId}/products
/accounts/{accountId}/sites/{siteId}/sippeers/{sippeerId}/products/origination/settings
/accounts/{accountId}/sites/{siteId}/sippeers/{sippeerId}/products/termination/settings
/accounts/{accountId}/sites/{siteId}/sippeers/{sippeerId}/products/messaging/features/sms
/accounts/{accountId}/sites/{siteId}/sippeers/{sippeerId}/products/messaging/features/mms
/accounts/{accountId}/sites/{siteId}/sippeers/{sippeerId}/products/messaging/features/mms/settings
/accounts/{accountId}/sites/{siteId}/sippeers/{sippeerId}/products/messaging/applicationSettings
/accounts/{accountId}/sites/{siteId}/sippeers/{sippeerId}/products/messaging/settings
/accounts/{accountId}/sites/{siteId}/inserviceNumbers
/accounts/{accountId}/sites/{siteId}/orders
/accounts/{accountId}/sites/{siteId}/orders/{orderid}
/accounts/{accountId}/sites/{siteId}/orders/{orderid}/tns
/accounts/{accountId}/sites/{siteId}/portins
/accounts/{accountId}/sites/{siteId}/totaltns

### CODE REVIEW

File: site/specs/numbers.json [hunk 1]
```diff
@@ -12424,58 +12423,105 @@
         "tags": [
           "/accounts"
         ],
-        "description": "<p>Add a site to the account</p>\n",
-        "operationId": "POST /accounts/{accountId}/sites",
+        "description": "Add a site to the account",
+        "summary": "Create Site",
```
Comment (mrmeyers99) [inline site/specs/numbers.json:12427]: Do we want to use the term site or sub account?

File: site/specs/numbers.json [hunk 2]
```diff
@@ -12489,55 +12535,55 @@
...
+                  "For a UC account": {
+                    "value": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>\n<SiteResponse>\n   <Site>\n      <Id>3013</Id>\n      <Name>Raleigh</Name>\n      <Description>Test Gateway</Description>\n      <CustomerName>BW</CustomerName>\n      <Address>\n         <HouseNumber>1600</HouseNumber>\n         <StreetName>PENNSYLVANIA</StreetName>\n         <StreetSuffix>AVE</StreetSuffix>\n         <PostDirectional>NW</PostDirectional>\n         <City>WASHINGTON</City>\n         <StateCode>DC</StateCode>\n         <Zip>20006</Zip>\n         <Country>US</Country>\n         <AddressType>Service</AddressType>\n      </Address>\n      <UcTrunkingConfiguration>\n         <Type>Seats</Type>\n         <UsageCategory>UC500</UsageCategory>\n      </UcTrunkingConfiguration>\n   </Site>\n</SiteResponse>"
                   }
                 }
               }
             }
           },
           "404": {
-            "description": "<p>the site id entered does not exist</p>\n",
+            "description": "The site was not found for entered id",
```
Comment (mrmeyers99) [inline site/specs/numbers.json:12581]:
```suggestion
            "description": "The site was not found for supplied id",
```

`entered` sounds like you're putting it into a field in a UI

File: site/specs/numbers.json [hunk 3]
```diff
@@ -12549,69 +12595,115 @@
...
-              "default": null
+              "type": "string"
             },
-            "example": null
+            "example": "1234567"
           },
           {
             "name": "siteId",
-            "description": "",
+            "description": "Site's ID for which information will be updated",
```
Comment (mrmeyers99) [inline site/specs/numbers.json:12614]:
```suggestion
            "description": "ID of the site to update",
```

File: site/specs/numbers.json [hunk 4]
```diff
@@ -12549,69 +12595,115 @@
...
         ],
         "requestBody": {
           "content": {
             "application/xml": {
+              "schema": {
+                "title": "CreateSite",
+                "properties": {
+                  "Name": {
+                    "description": "The name of the site. Max length restricted to 10 characters",
+                    "type": "string"
```
Comment (mrmeyers99) [inline site/specs/numbers.json:12631]: Should we use `maxLength` to restrict it to 10 characters?

https://swagger.io/docs/specification/data-models/data-types/#string

File: site/specs/numbers.json [hunk 5]
```diff
@@ -12549,69 +12595,115 @@
...
+                  "value": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>\n<Site>\n   <Name>Raleigh</Name>\n   <Description>SIP gateway</Description>\n   <CustomerName>BW</CustomerName>\n   <CustomerProvidedId>1234567890</CustomerProvidedId>\n   <Address>\n      <HouseNumber>1600</HouseNumber>\n      <StreetName>PENNSYLVANIA</StreetName>\n      <StreetSuffix>AVE</StreetSuffix>\n      <PostDirectional>NW</PostDirectional>\n      <City>WASHINGTON</City>\n      <StateCode>DC</StateCode>\n      <Zip>20006</Zip>\n      <Country>US</Country>\n      <AddressType>Service</AddressType>\n   </Address>\n   <UcTrunkingConfiguration>\n      <Type>Seats</Type>\n      <UsageCategory>UC500</UsageCategory>\n   </UcTrunkingConfiguration>\n</Site>"
                 }
               }
             }
           }
         },
         "responses": {
           "200": {
-            "description": "<p>the site id has been successfully updated</p>\n"
+            "description": "the site id has been successfully updated"
```
Comment (mrmeyers99) [inline site/specs/numbers.json:12686]:
```suggestion
            "description": "The site has been successfully updated"
```

File: site/specs/numbers.json [hunk 6]
```diff
@@ -12549,69 +12595,115 @@
...
           },
           "400": {
-            "description": "<p>Information essential to the site id is missing</p>\n",
+            "description": "Information essential to the site id is missing",
```
Comment (mrmeyers99) [inline site/specs/numbers.json:12689]:
```suggestion
            "description": "Information essential to the site is missing",
```

File: site/specs/numbers.json [hunk 7]
```diff
@@ -12722,92 +12810,188 @@
...
           }
         ],
         "requestBody": {
           "content": {
             "application/xml": {
+              "schema": {
+                "title": "CreateSipPeer",
+                "properties": {
+                  "PeerName": {
+                    "description": "Mandatory name for the SIP Peer (Max 10 chars)",
```
Comment (mrmeyers99) [inline site/specs/numbers.json:12845]: `maxLength`?

File: site/specs/numbers.json [hunk 8]
```diff
@@ -13703,77 +13875,99 @@
...
+                  "value": "<?xml version=\"1.0\" encoding=\"ISO-8859-1\" standalone=\"yes\"?>\n<SipPeerTerminationSettings>\n   <VoiceProtocol>SIP</VoiceProtocol>\n</SipPeerTerminationSettings>"
                 }
               }
             }
           }
         },
         "responses": {
           "201": {
-            "description": "<p>setting were successfully added</p>\n",
+            "description": "Setting were successfully added",
```
Comment (mrmeyers99) [inline site/specs/numbers.json:13952]:
```suggestion
            "description": "Settings were successfully added",
```

**Review by hamermike — APPROVED** [review]

File: site/specs/numbers.json [hunk 9]
```diff
@@ -12424,58 +12423,105 @@
...
+                  "Description" : {
+                    "description": "Customer provided description of the site",
+                    "type": "string"
+                  },
+                  "Address" : {
+                    "description": "Service Address for the site",
+                    "type": "object"
+                  },
+                  "CustomerProvidedId" : {
+                    "description": "Customer can provide an optional id (max 10 digits). Note that the customer can use the same id across multiple orders.",
```
Comment (mrmeyers99) [inline site/specs/numbers.json:12460]: We could use the `maximum` field instead of putting `max 10 digits` in the description

**Comment by hamermike** [comment]
@mrmeyers99 @safronenko23 I'd lean on whichever would be more clear to the user of the service. I'd guess that keeping the docs and API references consistent with the API would be the way to go for the moment as it's more closely aligned to the API endpoints. I'd question more on why we refer to them differently at all (that's quite confusing).

**Comment by mrmeyers99** [comment]
I don't know the history, but I'm guessing that we changed the names in the UI and never changed the API.  I think I agree that we shouldn't change this yet.  Sorry for the confusion @safronenko23

File: site/specs/numbers.json [hunk 10]
```diff
@@ -15285,42 +15412,40 @@
         "tags": [
           "/accounts"
         ],
-        "description": "<p>Retrieve count of Telephone numbers for the indicated site</p>\n",
-        "operationId": "GET /accounts/{accountId}/sites/{siteId}/totaltns",
+        "description": "Retrieves Site telephone numbers total count",
+        "summary": "Retrieve Site TNs count",
+        "operationId": "RetrieveSubAcoountTnsCount",
```
Comment (mrmeyers99) [inline site/specs/numbers.json:15417]:
```suggestion
        "operationId": "RetrieveSiteTnsCount",
```

Comment (safronenko23) [reply site/specs/numbers.json:15417]: ahh miss one :)

**Review by mrmeyers99 — APPROVED** [review]

**Pull request merged**